llvm/docs/ReleaseNotes.md

```diff
@@ -443,6 +443,10 @@ Changes to LLDB
           _regexp-display   -- Evaluate an expression at every stop (see 'h...
   ```
 
+  * Minidumps generated by LLDB now support 64 bit memory, the capture of TLS variables, and capturing multiple signals or exceptions. Additionally to support 64 bit memory, Minidumps are now paged to disk while being written.
+
+  * [New Core File API](https://lldb.llvm.org/python_api/lldb.SBSaveCoreOptions.html) with      greater control of data LLDB will capture into Core files.
```

Comment:
I would phrase it like:
```
A [new core file API] was added. This gives greater control...
```
And maybe add on the end `, relative to ...` with whatever the other way is. I guess the `process save-core` command.